csrankings.csv

```diff
@@ -6563,6 +6563,7 @@ Ghita Kouadri Mostéfaoui,University College London,http://www0.cs.ucl.ac.uk/peo
 Gholamreza Anbarjafari,University of Tartu,http://kodu.ut.ee/~sjafari,6gN67JAAAAAJ
 Gholamreza Ghassem-Sani,Sharif University of Technology,http://sina.sharif.ac.ir/~sani,NOSCHOLARPAGE
 Gholamreza Haffari,Monash University,http://users.monash.edu.au/~gholamrh,Perjx5EAAAAJ
+Gi-Soo Kim,UNIST,https://sdm.unist.ac.kr/,uW2qIcYAAAAJ

```

Comment:
Not in CS (in Industrial Engineering); please provide evidence that they meet inclusion criteria.

Reply:
Hello, 

We'd like to point out that newly added data are for new faculty members joining the AI graduate school of UNIST.
You can find the information in the link below:
https://aigs.unist.ac.kr/eng/pages/sub02_01.php?Track=c120
Also, we'd like to point out that they are therefore capable of advising PhD students in CS (regardless of their home department.)

Could this be the evidence for your comment on Gi-su Kim? 
If not, we'd like to reflect other requests first, except these two faculty members (Namhoon Lee and Gi-su Kim).
If necessary, we will make corresponding changes on our previous pull request.

Best regards,
Donguk